test/test_mlmodel.py

```diff
@@ -37,6 +37,23 @@ def test_properties():
     assert model_tf_adult.feature_input_order == exp_feature_order_adult
 
 
+def test_inverse_pipeline():
+    data_name = "adult"
+    data = DataCatalog(data_name)
+
+    model_tf_adult = MLModelCatalog(data, "ann")
+
+    transformed_data = model_tf_adult.perform_pipeline(data.raw)
+    detransformed_data = model_tf_adult.perform_inverse_pipeline(transformed_data)
+    # sklearn scales produces results of e.g. 38.9999999999 vs 39.0
+    detransformed_data[data.continous] = detransformed_data[data.continous].round(1)
```

Comment:
A better solution woule be something to use like pytest.approx instead of manually rounding somewhere in the middle of the test: https://docs.pytest.org/en/latest/reference/reference.html?highlight=approx#pytest.approx

Reply:
Turns out there is a even better solution: 

`assert_frame_equal(expected_data, detransformed_data)`